core/src/main/scala/kafka/server/ConfigHandler.scala

```diff
@@ -95,6 +96,19 @@ class TopicConfigHandler(private val logManager: LogManager, kafkaConfig: KafkaC
         .map(_ (0).toInt).toSeq //convert to list of partition ids
     }
   }
+  
+  def excludedConfigs(topic: String, topicConfig: Properties): Set[String] = {
+    val excludeConfigs: mutable.Set[String] = new mutable.HashSet[String]
```

Comment:
Yeah, we used to add `log.retention.ms` and `log.message.timestamp.difference.max.ms` here. Personally I feel the previous code and and the current one are both clear. I can change it back if you prefer the previous code.